variations/build-variations.mjs

```diff
@@ -42,7 +53,6 @@ async function buildVariation(source, variation) {
 		// remove unneeded resources
 		await fs.remove( destDir + '/sass');
 		await fs.remove( destDir + '/node_modules' );
```

Comment:
This is unrelated to the PR but it's the first time I'm trying out the `build:variations` script and I'm noticing the copy process feels a bit slow. I have a hunch the length of the copy process is impact by having to delete the node_modules folder for each variation, I wonder if there's an `fs` way of copying that can ignore certain files/folders to save some time.

Reply:
Yeah, something like that could be done to speed it up.  Prefer to do that in a separate PR as that can be completely under the hood.